glide.yaml

```diff
@@ -18,6 +18,8 @@ import:
 - package: k8s.io/gengo
 - package: github.com/ugorji/go
   version: v.1.1-beta
+- package: github.com/Azure/go-autorest
+  version: e14a70c556c8e0db173358d1a903dca345a8e75e
 - package: golang.org/x/tools
   subpackages:
   - imports
```

Comment:
Those are all the direct dependencies of the descheduler:

```sh
$ golist --package-path github.com/kubernetes-incubator/descheduler --imported
github.com/golang/glog
github.com/kubernetes-incubator/descheduler/cmd/descheduler/app
github.com/kubernetes-incubator/descheduler/cmd/descheduler/app/options
github.com/kubernetes-incubator/descheduler/pkg/api
github.com/kubernetes-incubator/descheduler/pkg/api/install
github.com/kubernetes-incubator/descheduler/pkg/api/v1alpha1
github.com/kubernetes-incubator/descheduler/pkg/apis/componentconfig
github.com/kubernetes-incubator/descheduler/pkg/apis/componentconfig/install
github.com/kubernetes-incubator/descheduler/pkg/apis/componentconfig/v1alpha1
github.com/kubernetes-incubator/descheduler/pkg/descheduler
github.com/kubernetes-incubator/descheduler/pkg/descheduler/client
github.com/kubernetes-incubator/descheduler/pkg/descheduler/evictions
github.com/kubernetes-incubator/descheduler/pkg/descheduler/evictions/utils
github.com/kubernetes-incubator/descheduler/pkg/descheduler/node
github.com/kubernetes-incubator/descheduler/pkg/descheduler/pod
github.com/kubernetes-incubator/descheduler/pkg/descheduler/scheme
github.com/kubernetes-incubator/descheduler/pkg/descheduler/strategies
github.com/kubernetes-incubator/descheduler/pkg/utils
github.com/spf13/cobra
github.com/spf13/pflag
github.com/ugorji/go/codec
k8s.io/api/core/v1
k8s.io/api/policy/v1beta1
k8s.io/apimachinery/pkg/api/errors
k8s.io/apimachinery/pkg/api/resource
k8s.io/apimachinery/pkg/apimachinery/announced
k8s.io/apimachinery/pkg/apimachinery/registered
k8s.io/apimachinery/pkg/apis/meta/v1
k8s.io/apimachinery/pkg/conversion
k8s.io/apimachinery/pkg/fields
k8s.io/apimachinery/pkg/labels
k8s.io/apimachinery/pkg/runtime
k8s.io/apimachinery/pkg/runtime/schema
k8s.io/apimachinery/pkg/runtime/serializer
k8s.io/apiserver/pkg/util/flag
k8s.io/apiserver/pkg/util/logs
k8s.io/client-go/kubernetes
k8s.io/client-go/listers/core/v1
k8s.io/client-go/plugin/pkg/client/auth
k8s.io/client-go/rest
k8s.io/client-go/tools/cache
k8s.io/client-go/tools/clientcmd
k8s.io/kubernetes/pkg/api/v1/resource
k8s.io/kubernetes/pkg/apis/core
k8s.io/kubernetes/pkg/apis/core/v1/helper
k8s.io/kubernetes/pkg/apis/core/v1/helper/qos
k8s.io/kubernetes/pkg/kubelet/types
k8s.io/kubernetes/plugin/pkg/scheduler/algorithm/priorities/util
```

Would be worth minimizing the content of the vendor cause only `k8s.io/kubernetes/pkg/kubelet/types` package is used although the vendor container entire tree of `k8s.io/kubernetes/pkg/kubelet`.

Though, not a blocker for the PR.